improver/metadata/probabilistic.py

```diff
@@ -155,6 +155,31 @@ def find_threshold_coordinate(cube):
     return threshold_coord
 
 
+def above_or_below(cube):
```

Comment:
Can I suggest calling this something a bit more descriptive, so that it makes sense in the contexts where it's called.  For example `probability_data_above_or_below`.